android/cli/commands/_buildModule.js

```diff
@@ -1907,10 +1886,10 @@ AndroidModuleBuilder.prototype.packageZip = function (next) {
 							return walk(file, path.join(parent, name));
 						}
 
-						let contents = fs.readFileSync(file).toString();
+						let contents = fs.readFileSync(file);
```

Comment:
I don't understand why you removed the `.toString()`. `dest.append()` will accept a string.

On a side note, I've started moving away from `fs.readFileSync(file).toString()` and towards `fs.readFileSync(file, 'utf8')`.

Reply:
For non-markdown files (like images), we do not want to generate HTML from it and therefore only need the string-representation for valid markdown files. So in this case, we cannot use `readFileSync(file, 'utf8')` either as we still need its raw buffer for other files than markdown.

Reply:
That's a pretty darn good reason!